react-components/src/data/edit/Comment.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import Typography from "@material-ui/core/Typography";
+import ListItem from "@material-ui/core/ListItem";
+import DeleteIcon from "@material-ui/icons/Delete";
+import Fab from "@material-ui/core/Fab";
+import Divider from "@material-ui/core/Divider";
+
+const RETRACTED = "Retracted";
+
+class Comment extends Component {
```

Comment:
Looks like all the Comment related stuff under `/data/comments` is still here under `/data/edit` 👀